### PR TITLE
qa/tests: initial checkin for nautilus-p2p

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by yuriw** [comment]
@neha-ojha 
Open questions for `nautilus-p2p-stress-split`:

- what do we do about `1.1-pg-log-overrides -> ../../luminous-x/parallel/1.1-pg-log-overrides/`, use similar but from `mimic-x` ?

- same question about `objectstore/bluestore.yaml -> ../../../../../objectstore/bluestore.yaml`

**Comment by neha-ojha** [comment]
> @neha-ojha
> Open questions for `nautilus-p2p-stress-split`:
> 
> * what do we do about `1.1-pg-log-overrides -> ../../luminous-x/parallel/1.1-pg-log-overrides/`, use similar but from `mimic-x` ?


> * same question about `objectstore/bluestore.yaml -> ../../../../../objectstore/bluestore.yaml`

**Comment by neha-ojha** [comment]
> > @neha-ojha
> > Open questions for `nautilus-p2p-stress-split`:
> > 
> > * what do we do about `1.1-pg-log-overrides -> ../../luminous-x/parallel/1.1-pg-log-overrides/`, use similar but from `mimic-x` ?

we should symlink short_pg_log.yaml in 1.1-pg-log-overrides both in luminous-x and mimic-x to qa/overrides/short_pg_log.yaml 
> 
> > * same question about `objectstore/bluestore.yaml -> ../../../../../objectstore/bluestore.yaml`

this should be symlinked to qa/objectstore

**Comment by yuriw** [comment]
@neha-ojha  incorporated comments, thx!

File: qa/suites/upgrade/nautilus-p2p/nautilus-p2p-stress-split/2-partial-upgrade/firsthalf.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+   restart : osd.0,1,2,3
+tasks:
+- install.upgrade:
+    osd.0:
+- print: "**** done install.upgrade osd.0"
+- ceph.restart:
+    daemons: [mon.a,mon.b,mon.c,mgr.x,osd.0,osd.1,osd.2,osd.3]
+    mon-health-to-clog: false
+- print: "**** done ceph.restart 1st half"
+- exec:
```
Comment (neha-ojha) [inline qa/suites/upgrade/nautilus-p2p/nautilus-p2p-stress-split/2-partial-upgrade/firsthalf.yaml]: we don't need lines 14-18

File: qa/suites/upgrade/nautilus-p2p/nautilus-p2p-stress-split/5-finish-upgrade.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+tasks:
+- install.upgrade:
+    osd.4:
+    client.0:
+- ceph.restart:
+    daemons: [osd.4, osd.5, osd.6, osd.7]
+    wait-for-healthy: false
+    wait-for-osds-up: true
+- exec:
```
Comment (neha-ojha) [inline qa/suites/upgrade/nautilus-p2p/nautilus-p2p-stress-split/5-finish-upgrade.yaml]: we don't need lines 9-13

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2019-03-26_15:31:17-upgrade:nautilus-p2p-nautilus-distro-basic-smithi/

**Pull request merged**